rancher.py

```diff
@@ -350,6 +356,7 @@ def reload_schema(self):
 
     def by_id(self, type, id, **kw):
         id = str(id)
+        type = convert_type_name(type)
```

Comment:
Don't overwrite `type` name is something else incase we need to use this string in the future.